etc/kayobe/ansible/requirements.yml

```diff
@@ -14,18 +12,19 @@ collections:
     version: 1.1.0
 roles:
   - src: stackhpc.vxlan
-  - name: ansible-lockdown.ubuntu22_cis
-    src: https://github.com/ansible-lockdown/UBUNTU22-CIS
-    version: 1.4.1
+  - name: ansible-lockdown.ubuntu24_cis
+    src: https://github.com/ansible-lockdown/UBUNTU24-CIS
+    version: 1.0.0
   - name: ansible-lockdown.rhel9_cis
     src: https://github.com/ansible-lockdown/RHEL9-CIS
-    version: 1.3.1
+    version: v1.3.4
```

Comment:
There is a new version of ansible-lockdown.rhel9_cis out now.

Reply:
Updated